xml/common_intro_typografie_i.xml

```diff
@@ -11,20 +11,59 @@
  <title>Documentation Conventions</title>
 
  <info>
-<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-    <dm:maintainer>fs</dm:maintainer>
-    <dm:status>editing</dm:status>
-    <dm:deadline/>
-    <dm:priority/>
-    <dm:translation>no</dm:translation>
-    <dm:languages/>
-</dm:docmanager>
-</info>
-<para>
-  The following typographical conventions are used in this manual:
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>fs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  The following admonition elements and typographical conventions are used
+  in this documentation:
  </para>
 
  <itemizedlist mark="bullet" spacing="normal">
+  <warning>
+   <para>
+    Vital information to be aware of before proceeding. Warns you about
```

Comment:
For the <warning/>, I have changed this to: Vital information you must be aware of before proceeding. For <important/>, I would use: Important information you should be aware of before proceeding.